Uebungsaufgaben/OOP4/tests/CardTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers Card
+ */
+class CardTest extends TestCase
+{
+    /** @var Card */
+    private $card;
+    /** @var PHPUnit_Framework_MockObject_MockObject|Color */
+    private $color;
+
+    protected function setUp()
+    {
+        $this->color = $this->createMock(Color::class);
+        $this->card = new Card($this->color);
+    }
+
+    public function testTurn() {
```

Comment:
testCardCanBeTurned. 
=> Sprechende Testmethoden (gilt überall, bitte nochmals überarbeiten).

Dann kannst du phpunit --testdox in der konsole aufrufen zum testen.